GCMessengerSDKSample/Podfile

```diff
@@ -12,12 +12,28 @@ use_frameworks! :linkage => :static
 install! 'cocoapods',
          :deterministic_uuids => false
 
-source 'https://github.com/MyPureCloud/gc-sdk-specs'
+DX_CORE="../../../BitBucket/mobiledx-ios-sdk-core/GenesysCloudCore-Dev.podspec"
```

Comment:
You need to reset the Podfile to master version.

Reply:
how do i do that?

Reply:
After the merge, create a branch on master to delete the unnecessary part, and to point the right podspec. Then create a PR.
But before that you need to build the RC, otherwise it won't compile.

Reply:
We need a better process for this. It's messy, I know.

Reply:
ok.. after a chat with Eliza and Kalman, we will run the official already instead of building an RC. 
so the steps are going to be (please approve those)
1. please approve this PR
2. I will merge it
3. I will create the build using jenkins.
4. I will restore this podfile to the original 'master' one.

Deal?

Reply:
First restore the Podfile, then create build from Jenkins.